webpack/ForemanTasks/Components/TaskDetails/Components/Errors.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const Errors = ({ ...props }) => {
+  const { failedSteps, executionPlan } = props;
+  if (!executionPlan)
+    return (
+      <Alert type="error">{__('Execution plan data not available ')}</Alert>
+    );
+  if (!failedSteps.length)
+    return <Alert type="success">{__('No errors')}</Alert>;
+  return (
+    <div>
+      {failedSteps.map((step, i) => (
+        <Alert type="error" key={i}>
+          <span className="param-name">{__('Action')}:</span>
+          <span className="param-value">
+            <pre>{step.action_class}</pre>
+          </span>
+          <span className="param-name">{__('Input')}:</span>
+          <span className="param-value">
+            <pre>{step.input}</pre>
+          </span>
+          <span className="param-name">{__('Output')}:</span>
+          <span className="param-value">
+            <pre>{step.output}</pre>
+          </span>
+          {step.error ? (
```

Comment:
Wouldn't `{step.error && (<React.Fragment>...</React.Fragment>)}` do the same thing without the dangling empty string?

Reply:
Youre right, thanks!

Reply:
There are still some places where this could be applied

Reply:
I changed them as well, thanks